Yosemite/YosemiteTests/Stores/OrderStoreTests.swift

```diff
@@ -563,6 +563,143 @@ final class OrderStoreTests: XCTestCase {
         wait(for: [expectation], timeout: Constants.expectationTimeout)
     }
 
+    func test_optimistic_update_order_customer_note_correctly() {
+        // Given
+        let orderStore = OrderStore(dispatcher: dispatcher, storageManager: storageManager, network: network)
+        let expectedCustomerNote = ""
+
+        let originalOrder = sampleOrder().copy(customerNote: "Updated!")
```

Comment:
Super nit: should the notes be the other way around? This looks confusing to me 😅